### DISCUSSION TITLE
unfair distribution of rewards

### DISCUSSION THREAD

**Discussion by kantarbozan** [IBAX-io/go-ibax#527] [Rewards]
I find the A and B planning of the rewards wrong. Those who fail to enter the top thousand and complete all of the quests and only complete a few of the quests are equally rewarded.

This rule should be updated.
It is also frequently discussed in telegram channels, but we could not reach anyone. ibxc team, we are waiting for your help, thank you

**Comment by Abdullahoda** [comment]
Evet haklısın ayrıca her gün ibax ödülleri dagıtılacak tı ama dagıtılmadı..

**Reply by kantarbozan** [reply]
katılıyorum kesinlikle ekibin duyması lazım

**Comment by hes91** [comment]
O zaman daha haksızlık olur.A ve B olarak gruplandırılması iyi fikir.Kim daha fazla emek harcarsa daha çok IBXC alacak gayet adaletli bence.Emek göstermeden bir şeyler talep etmek daha haksızlık olur.

**Reply by kantarbozan** [reply]
herkes puanıyla orantılı alsın istiyoruz
ilk bin e gireceğini düşünüyo musun 40 bin telegramda var sadece 
ilk bine giremezsen 2 3 tane ibxc düşecek adam başı
ve rasgele 7 kişiye a gurubuna ayrılanın yarısı gidecek aşırı saçma
